BRANDING.MD

```diff
@@ -0,0 +1,24 @@
+## Knative Branding Guidelines
+
+
+The Knative logo is made available here with the following guidelines:
```

Comment:
please clarify if "logo" means just the picture or does it also include the name "Knative" ?

Reply:
I'm guessing it's just the picture, so can we get text about the name too ?

Reply:
Specific name usage would want to go to Steering for review/approval. I'll add some text on this to be clear!

Reply:
Please don't make it so that everyone has to go to the SC each time - that would be a huge bottleneck. There should be some general guidelines (with examples) and then only require people to go to the SC if they deviate from those.